pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "Cython",
+    "Cython<3.0.0",
```

Comment:
Hmm. I've never used `Cython` before, but it's a valid pin so ok  👍

Reply:
yeah had to pin to a version before 3.0, which was a major change. Our code wont compile with 3 and higher